src/main.js

```diff
@@ -118,49 +121,48 @@ const startWithMBTiles = (mbtilesFile) => {
       }
       const bounds = info.bounds;
 
-      const styleDir = path.resolve(__dirname, "../node_modules/tileserver-gl-styles/");
+      const styleDir = path.resolve(__dirname, '../node_modules/tileserver-gl-styles/');
 
       const config = {
-        "options": {
```

Comment:
We should use standard Eslint & Prettier configs for auto cleanup of styling...

Reply:
I ran eslint on this, but could use suggestions on defaults

Reply:
Same as below, these keys could actually have the quotes removed entirely since they're js rather than JSON.

Reply:
Also, I'm a fan of eslint/prettier; it ends up touching most lines of the project so I'd suggest getting anything in-progress merged before running it to reduce the amount of merge conflicts.